octopoes/config/settings.py

```diff
@@ -21,7 +21,7 @@ class Settings(BaseSettings):
 
     # External services settings
     dsn_katalogus: str = "http://katalogus:8000/"
-    dsn_xtdb: str = "http://xtdb:3000/"
+    dsn_xtdb: str = "http://xtdb:3000/_xtdb"
```

Comment:
I want to suggest the following:
- Use a more strict type than `str`, such as e.g. Pydantic's `Any(HTTP)Url`. You get validation and documentation for free
- The above point makes the `dsn_` prefix redundant

This goes for every `dsn_*` setting

Reply:
Created a separate issue for this:

https://github.com/minvws/nl-kat-octopoes/issues/83

Reply:
Created a separate issue for this:

https://github.com/minvws/nl-kat-octopoes/issues/83